cmd/helm/helm_test.go

```diff
@@ -264,6 +263,7 @@ func TestTLSFlags(t *testing.T) {
 				TLSCaCertFile:           home.TLSCaCert(),
 				TLSCertFile:             home.TLSCert(),
 				TLSKeyFile:              home.TLSKey(),
+				SystemPluginsDir:        "/usr/share",
```

Comment:
Helm is used and developed on Windows. We need tests passing there so the plugin directory needs to be right for different target platforms.

This does mean, we need windows support in the system wide plugins.